cmd/traffic/cmd/manager/managerutil/envconfig.go

```diff
@@ -0,0 +1,42 @@
+package managerutil
+
+import (
+	"context"
+
+	"github.com/sethvargo/go-envconfig"
+)
+
+type Env struct {
+	User        string `env:"USER,default="`
+	AgentImage  string `env:"TELEPRESENCE_AGENT_IMAGE,default="`
+	AgentPort   int32  `env:"TELEPRESENCE_AGENT_PORT,default=9900"`
+	Namespace   string `env:"MANAGER_NAMESPACE,default="`
```

Comment:
I don't like having short envconfig field names like this.  If we thought it was worth it to put `MANAGER_` in the env-var name, then surely it's worth it to put `Manager` in the field name.